src/Tools/dotnet-dump/Analyzer.cs

```diff
@@ -56,7 +56,16 @@ public async Task<int> Analyze(FileInfo dump_path, string[] command)
                     target = DataTarget.LoadCoreDump(dump_path.FullName);
                 }
                 else if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) {
-                    target = DataTarget.LoadCrashDump(dump_path.FullName, CrashDumpReader.ClrMD);
+                    try
+                    {
+                        target = DataTarget.LoadCoreDump(dump_path.FullName);
+                    }
+                    catch { }
```

Comment:
We usually don't catch all exceptions.  Exceptions like null ref, etc. that shouldn't be catch and terminates dotnet-dump. Catch only the exceptions expected for this failure. 

Nit: the {} should be on their own lines.